### PR TITLE
feat: add reusable listing card styles

### PR DESCRIPTION
**Pull request by WillKocur**
Adds reusable styles for listing card components. To be used on home, resources, indicators and search page.

### CODE REVIEW

**Comment by WillKocur** [comment]
@nym1k Just flagging that I've removed some styling that is still being used on this branch, but it won't be being used moving forwards I just didn't want to put the template changes in this PR. E.g. search-page__results-body won't be used going forward.

File: app/assets/stylesheets/components/icon.scss [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+  display: inline-block;
+}
+
 @include b(icon) {
-  @include m(left)  { margin-right: vr(.75); }
-  @include m(right) { margin-left: vr(.75);  }
+  @include m(left)  { margin-right: vr(1); }
+  @include m(right) { margin-left: vr(1);  }
   @include m(sub)   { vertical-align: sub;   }
```
Comment (nym1k) [inline app/assets/stylesheets/components/icon.scss:12]: tiny thing but the spacing looks inconsistent here between the ; and the closing bracket.

File: app/assets/stylesheets/variables/typography.scss [hunk 2]
```diff
@@ -137,12 +137,19 @@ $font-book: (
...
-    font-weight: 500,
-    line-height: 1.5,
+    font-family: Merriweather,
+    font-size: 1.375rem,
+    font-weight: 700,
+    line-height: 1.4,
     color: $prussian-blue
   ),
+  search-result-link: (
+    font-family: Open Sans,
```
Comment (nym1k) [inline app/assets/stylesheets/variables/typography.scss:147]: I think we should put font family names with spaces in to single quotes (e.g. 'Open Sans')

Comment (nym1k) [reply app/assets/stylesheets/variables/typography.scss:147]: Just to note, i've already done this for the other fonts on another branch

**Review by nym1k — CHANGES_REQUESTED** [review]
Very minor changes required, then good to merge

File: app/assets/stylesheets/components/icon.scss (hunk 1 above)
Comment (WillKocur) [reply app/assets/stylesheets/components/icon.scss]: It is consistent in that it increases by one space each time.... ;) Fixed now.

**Comment by WillKocur** [comment]
@nym1k Changes made - thanks for the speedy review!

**Review by nym1k — APPROVED** [review]

**Pull request merged**